### PR TITLE
Adding test to compare output of csg_reupdate

### PR DESCRIPTION
**Pull request by junghans**
Part of votca/votca#71

### CODE REVIEW

**Review by JensWehner — APPROVED** [review]
works for me, but why is the example so large?

**Comment by junghans** [comment]
Good point, let me shrink that a bit!

**Comment by junghans** [comment]
@JensWehner: Done

File: src/tools/CMakeLists.txt [hunk 1]
```diff
@@ -21,3 +21,13 @@ foreach(PROG csg_reupdate csg_map csg_dump csg_property csg_resample csg_stat cs
     set_tests_properties(${PROG}Help PROPERTIES LABELS "csg;tools;votca")
   endif(ENABLE_TESTING)
 endforeach(PROG)
+
+if(ENABLE_TESTING)
```
Comment (junghans) [inline src/tools/CMakeLists.txt:25]: @JoshuaSBrown that is the part you want to look at.

**Pull request merged**